doc/onboarding.md

```diff
@@ -86,6 +86,10 @@ onboarding session.
       `semver-major` label
     * When adding a `semver-*` label, add a comment explaining why you're adding
       it. Do it right away so you don't forget!
+  * Please add the `author-ready` label for PRs where:
+    * the CI has been started (not necessarily finished),
+    * no outstanding review comments exist and
+    * at least one collaborator approved the PR.
```

Comment:
I personally would not add the label if it's semver-major and has not received 2 TSC approval yet...probably just a different interpretation of the label though.

Reply:
I have no strong opinion either way. It was just my current interpretation. Because I feel the author is ready as soon as a collaborator approved it. We have to make sure it gets enough LGs one way or the other and we know that we have to trigger the TSC in case some are missing.

Shall I change it?

Reply:
@BridgeAR We can revisit that later, probably need to update the label description as well if we are going to change its meaning